x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/original_scenarios.md

```diff
@@ -0,0 +1,264 @@
+# Installation and Upgrade of Prebuilt Rules
+
+## Scenarios
+
+### Notifications
+
+#### **Scenario: No callout messages are displayed when there are no pending installs/updates**
+
+```Gherkin
+Given user has the latest version of prebuilt rules <prebuilt_rules_status>
+When user navigates to the Rule Management page
+Then no callout message is displayed for <prebuilt_rules_status> rules
+And no badges with number of available rules to install/update are displayed
+
+Examples:
+  | prebuilt_rules_status |
+  |            to install |
+  |             to update |
+```
+
+#### **Scenario: Callout message is displayed when there are new prebuilt rules available to install**
+
+```Gherkin
+Given user already has 8.x prebuilt rules installed
+And there are new prebuilt rules available to install
+When user navigates to the Rules Management Page
+Then user should see a callout message to install new prebuilt rules
+And the number of new rules available to install should be displayed on the +Add Elastic Rules link
+```
+
+#### **Scenario: Callout message is displayed when there are new updates on already installed prebuilt rules**
+
+```Gherkin
+Given user already has 8.x prebuilt rules installed
+And there are new updates available for those prebuilt rules
+When users navigate to the Rules Management Page
+Then users should see an update callout message
+And the number of outdated rules should be displayed on the Rules Updates tab
+```
+
+#### **Scenario: User is notified of available prebuilt rules to install when a rule is deleted**
+
+```Gherkin
+Given user has the latest version of prebuilt rules installed
+When user navigates to Rules Management Page
+And user deletes some prebuilt rules
+Then user should see a callout message with the same amount of prebuilt rules ready to install
+And the updated number of available rules to install should be displayed on the +Add Elastic Rules link
+```
+
+#### **Scenario: User is notified that all available rules have been installed in the Add Elastic Rules**
+
+```Gherkin
+Given user has all available rules installed
+When user navigates to the Add Elastic Rules Page
+Then user should see a message indicating that all available rules have been installed
+And user should see a CTA that leads to the Rules Management Page
+And user should navigate back to Rules Management Page when clicking on the CTA
+```
+
+#### **Scenario: User is notified that all installed rules are up to date in the Rule Updates tab**
+
+```Gherkin
+Given user has all available rules installed
+When user navigates to the Rule Update
+Then user should see a message indicating that all installed rules are up to date
+```
```

Comment:
I deleted this one and instead, I suggest hiding the Rule Updates tab:

#### **Scenario: User is NOT notified when all prebuilt rules are installed and up to date**

**Coverage**: 1 e2e test with mock rules + 1 integration test.

```Gherkin
Given all the latest prebuilt rules are installed in Kibana
When user opens the Rule Management page
Then user should NOT see a CTA to install prebuilt rules
And user should NOT see a number of rules available to install
And user should NOT see a CTA to upgrade prebuilt rules
And user should NOT see a number of rules available to upgrade
And user should NOT see the Rule Updates table
```